workflow-trigger

```diff
@@ -1 +1 @@
-2022-12-02T20:00:00Z
+2022-12-07T20:00:00Z
```

Comment:
[trigger all workflows to run](https://github.com/airqo-platform/AirQo-api/pull/1216/commits/1783dca81c59c59d5fd004467a512085794fc5f5)